setup.sh

```diff
@@ -30,6 +30,7 @@ set -e
 # Check if sudo is available
 if command -v sudo >/dev/null 2>&1; then
```

Comment:
Not blocking this PR, but it seems like the two branches should be the same and the deviation is what broke us. It would be good if we can remove the branch, to the same effect as:

```bash
(sudo bash || bash) <<'EOF'
apt update && \
...
EOF
```

Reply:
That's a good idea. I modified as suggested. Thanks